elfpy/bots/get_env_args.py

```diff
@@ -0,0 +1,81 @@
+"""Get environment arguments for bots"""
+import os
+from dataclasses import dataclass
+from enum import Enum
+
+from elfpy import DEFAULT_LOG_MAXBYTES
+
+
+class LogLevel(Enum):
```

Comment:
Can you unify this with the existing solution? https://github.com/delvtech/elf-simulations/blob/main/elfpy/utils/outputs.py#L463

Reply:
ya i'll make a logging.py soon enuff